python/FineCollectionService/.env

```diff
@@ -0,0 +1,2 @@
+VEHICLE_REGISTRATION_ADDRESS=http://localhost:6002
+LICENSE_KEY=HX783-K2L7V-CRJ4A-5PN1G
```

Comment:
I can't recall where this environment variable is in use. Are you sure it's needed?

Reply:
The first one is used to locate the vehicle registration. The second is used on exercise 1 although you don't notice that since we don't have a call that triggers the error condition in the fine calculator.